index.html

```diff
@@ -0,0 +1,445 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="js/index.js"></script>
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Technical Documentation By Nwachan and Olivier</title>
+  </head>
+  <body>
+    <nav id="navbar">
+      <header>Technical Documentation By Nwachan and Olivier</header>
+      <ul>
+        <li><a class="nav-link" href="#Introduction">Introduction</a></li>
+        <li>
+          <a class="nav-link" href="#What_you_should_already_know"
+            >What you should already know</a
+          >
+        </li>
```

Comment:
HTML header changed to HTML5 Header

Reply:
kindly it is better to always start with courteous words like please or kindly. above all, you have to make understand the reason that makes you ask for the change, for example, here you should say ` it is advisable to use semantics so that  the browser knows what to read accurately`